src/ng-select/ng-select.component.ts

```diff
@@ -785,7 +786,9 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
         }
 
         if (this.multiple) {
-            this.unselect(this.itemsList.lastSelectedItem)
+            if (this.clearOnBackspace) {
```

Comment:
why this is applicable only for multiple? I think input should work for single select too

Reply:
at first I thought of only adding what needed in the issue, but yes sure should be applicable for single select too